### PR TITLE
Viresh_Assignment_31_Done_20Mins

### PR DESCRIPTION
**Pull request by VireshJJun20**
Please review and comment. Thanks.

image

### CODE REVIEW

File: src/vireshJJun20/Assignment31/SumOfNumbersExceptionHandling.java [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+			return inputArr1;
+		else
+			return null;
+	}
+	
+	// This method accepts int array and will display the sum of each array element. 
+	void displaySumOfNumbers(int[] inputArr) {
+		int sum=0;
+		for (int index = 0; index < inputArr.length; index++) {
+			sum += inputArr[index];
```
Comment (DeavinaSJun20) [inline src/vireshJJun20/Assignment31/SumOfNumbersExceptionHandling.java:50]: here you can combine both for loops and calculate sum and print the values in a single loop.

**Review by DeavinaSJun20 — CHANGES_REQUESTED** [review]
Comment given

File: src/vireshJJun20/Assignment31/SumOfNumbersExceptionHandling.java (hunk 1 above)
Comment (VireshJJun20) [reply src/vireshJJun20/Assignment31/SumOfNumbersExceptionHandling.java:50]: Yes, I will update it. Thanks.

**Review by KrishnaKTechnocredits — APPROVED** [review]
Code looks good to me.

**Pull request merged**